mscore/editpitch.cpp

```diff
@@ -34,7 +34,7 @@ EditPitch::EditPitch(QWidget *parent)
       setObjectName("EditPitchNew");
       setupUi(this);
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
-      tableWidget->setCurrentCell(4, 0);                    // select centre C by default
+      tableWidget->setCurrentCell(tableWidget->rowCount()-1-5, 0);                    // select centre C by default
```

Comment:
I hope I got this one right, any idea how to check?

Reply:
Or should we here, to be safe, search for the cell containing "C 4" (untranslated)?